tests/data/definitions3/meta/meta_indicators_more_data.yaml

```
@@ -0,0 +1,6 @@
+- exclude:
+    allowed_values: ['A', 'B']
+- number:
+    allowed_values: [1.0, 2.0, 3.0, 4.0]
+- string:
+    allowed_values: ['foo', nan]

```

"nan" and the Python object `None` often get confused by the yaml-parsing - I suggest to only work with actual strings for now.
```suggestion
    allowed_values: ['foo', 'bar']

```